feat(dcgw): Managed cache 1.1 (#5051)
Kong/developer.konghq.com: app/_includes/gateway/managed-cache-recommendation-note.md

{:.neutral}
> **Review managed cache size recommendations**
>

> > Before you create a managed cache, review [sizing recommendations](/dedicated-cloud-gateways/managed-cache/) to ensure your managed cache is the correct size. 

Reviewer: ```suggestion
> Before you create a managed cache, review [sizing recommendations](/dedicated-cloud-gateways/managed-cache/) to select the correct size. 
```
Slight rephrase to adjust order of operations - this sounds more like a "before" action + less repetition.
